### DISCUSSION TITLE
TXT Domain Secret on Epic Games

### DISCUSSION THREAD

**Discussion by LaymGlitched** [community/community#108633] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So, in my organization, I have a github pages website setup and working.
I'm trying to verify my game on epic games and to do so, i need to verify ownership of my website, Epic Games Developer portal ask me to put the TXT Domain Secret on the Domain to verify it.

![image](https://github.com/community/community/assets/65513977/53b7c368-3062-4259-8d51-f6ccc062b46c)

As far as i know, I am unable to do this on Github Pages. Any and all help needed,
Thank you.

**Comment by yoannchaudet** [comment]
Hello,

The `github.io` domain is not owned by you but GitHub. We don't allow you to create arbitrary DNS (e.g. `TXT`) records on it unfortunately.
